tsl/test/shared/expected/ordered_append_join-15.out

```diff
@@ -69,16 +70,14 @@ FROM (
 QUERY PLAN
  Nested Loop (actual rows=4 loops=1)
    ->  Values Scan on "*VALUES*" (actual rows=2 loops=1)
-   ->  Materialize (actual rows=2 loops=2)
-         ->  Subquery Scan on l (actual rows=2 loops=1)
-               ->  Limit (actual rows=2 loops=1)
-                     ->  Result (actual rows=2 loops=1)
-                           ->  Custom Scan (ChunkAppend) on metrics (actual rows=2 loops=1)
-                                 Order: metrics."time" DESC
-                                 ->  Index Scan using _hyper_X_X_chunk_metrics_time_idx on _hyper_X_X_chunk (actual rows=2 loops=1)
-                                 ->  Index Scan using _hyper_X_X_chunk_metrics_time_idx on _hyper_X_X_chunk (never executed)
-                                 ->  Index Scan using _hyper_X_X_chunk_metrics_time_idx on _hyper_X_X_chunk (never executed)
-(11 rows)
+   ->  Limit (actual rows=2 loops=2)
```

Comment:
This plan change looks a little mysterious to me. Since this query can only use the nested loop anyway because of the lateral, maybe we can re-enable it for this query only?

Reply:
You know I'll revert this ordered append join stuff because looks like this test is not flaky (at least I didn't noticed it).